tests/foreign_object/test_agnostic_order_trimjoin.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+from __future__ import unicode_literals, print_function, absolute_import
+
+from django.apps import apps
+from django.conf import settings
+from django.db.migrations.autodetector import MigrationAutodetector
+from django.db.migrations.loader import MigrationLoader
+from django.db.models.deletion import CASCADE
+from django.test.testcases import TestCase
+from .models import Customer, Contact, PhoneNumber, Address
+
+
+
+class TestLookupQuery(TestCase):
+    fixtures = ["customers.json"]
```

Comment:
We prefer the `setUpTestData()` method rather than using fixtures as the latter isn't as maintainable.